invenio_accounts_rest/views.py

```diff
@@ -81,3 +237,67 @@ def get(self):
             ).all()
         else:
             return User.query.all()
+
+
+blueprint.add_url_rule(
+    '/roles',
+    view_func=ListRolesResource.as_view(
+        ListRolesResource.view_name
+    )
+)
+
+blueprint.add_url_rule(
+    '/roles/<string:role_id>',
+    view_func=RoleResource.as_view(
+        RoleResource.view_name
+    )
+)
+
+
+blueprint.add_url_rule(
+    '/roles/<string:role_id>/users/<string:user_id>/assign',
+    view_func=AssignRoleResource.as_view(
+        AssignRoleResource.view_name
+    )
+)
+
+
+blueprint.add_url_rule(
+    '/roles/<string:role_id>/users/<string:user_id>/unassign',
```

Comment:
DELETE '/roles/string:role_id/users/string:user_id'
would be more REST (no verb in the URL).